manala.supervisor/tests/0100_install.goss.yml

```diff
@@ -1,13 +1,15 @@
 ---
 
-package:
-  supervisor:
-    installed: true
-
 command:
+{{ if eq .Env.DISTRIBUTION_RELEASE "wheezy" "jessie" }}
   dpkg -s supervisor:
     exit-status: 0
     stdout:
       - "/^Version: [\\d\\.-]+manala\\d+~{{ .Env.DISTRIBUTION_RELEASE }}\\d+$/"
-  supervisord --version:
-    exit-status: 0
+{{ else }}
+      - "Version:"
+{{ end }}
+package:
```

Comment:
You should keep package present assert first

Reply:
yes, and put condition on the right place =P